app/assets/stylesheets/views/_transition-countdown.scss

```diff
@@ -0,0 +1,76 @@
+$dark-blue-2: #003078;
+
+.Transition-countdown {
```

Comment:
Is this capitalised for a reason? We don't capitalise GOVUK for example?

Reply:
I don't think so. @DilwoarH is it ok for me to change?